openshift-client/src/main/java/io/fabric8/openshift/client/OpenShiftCreateOnlyResourceOperationsImpl.java

```diff
@@ -39,53 +35,21 @@ public OpenShiftCreateOnlyResourceOperationsImpl(OkHttpClient client, Config con
   public OpenShiftCreateOnlyResourceOperationsImpl(OperationContext context, String apiGroupName, String apiGroupVersion, String plural, Class<O> responseClass) {
     super(context.withApiGroupName(apiGroupName)
       .withApiGroupVersion(apiGroupVersion)
-      .withPlural(plural));
+      .withPlural(plural), apiGroupName, apiGroupVersion, plural, responseClass);
     this.subjectAccessApiGroupName = apiGroupName;
     this.subjectAccessApiGroupVersion = apiGroupVersion;
     this.plural = plural;
     this.responseClass = responseClass;
   }
 
-  @Override
-  public O create(I... resources) {
-    try {
-      if (resources.length > 1) {
-        throw new IllegalArgumentException("Too many items to create.");
-      } else if (resources.length == 1) {
-        return handleCreate(resources[0], responseClass);
-      } else if (getItem() == null) {
-        throw new IllegalArgumentException("Nothing to create.");
-      } else {
-        return handleCreate(getItem(), responseClass);
-      }
-    } catch (ExecutionException | IOException e) {
-      throw KubernetesClientException.launderThrowable(e);
-    } catch (InterruptedException ie) {
-      Thread.currentThread().interrupt();
-      throw KubernetesClientException.launderThrowable(ie);
-    }
-  }
-
-  @Override
-  public O create(I item) {
-    try {
-      return handleCreate(item, responseClass);
-    } catch (ExecutionException | IOException e) {
-      throw KubernetesClientException.launderThrowable(e);
-    } catch (InterruptedException ie) {
-      Thread.currentThread().interrupt();
-      throw KubernetesClientException.launderThrowable(ie);
-    }
-  }
-
   @Override
   public OpenShiftCreateOnlyResourceOperationsImpl<I, O> inNamespace(String namespace) {
-    this.namespace = namespace;
     return new OpenShiftCreateOnlyResourceOperationsImpl<>(context.withNamespace(namespace), subjectAccessApiGroupName, subjectAccessApiGroupVersion, this.plural, responseClass);
   }
 
-  public I getItem() {
-    return (I) context.getItem();
+  @Override
+  public boolean isResourceNamespaced() {
+    return true;
```

Comment:
Which class do we use for cluster scoped resources?

Reply:
We're using `io.fabric8.openshift.client.dsl.internal.OpenShiftCreateOnlyResourceNonNamespaceOperationsImpl` in case resource is cluser scoped

Reply:
Umm, I think these classes are redundant. We should replace these by [CreateOnlyResourceOperationsImpl](https://github.com/fabric8io/kubernetes-client/blob/master/kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/CreateOnlyResourceOperationsImpl.java) and [LocalCreateOnlyResourceOperationsImpl.java](https://github.com/fabric8io/kubernetes-client/blob/master/kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/LocalCreateOnlyResourceReviewOperationsImpl.java)

Reply:
:exploding_head:

Reply:
Sorry for the confusion, I'll create a follow-up PR to get rid of this duplication.